libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -188,18 +188,36 @@ export function trigger({ element, event, type } = {}) {
 
 export const yieldToMain = () => new Promise((resolve) => { setTimeout(resolve, 0); });
 
-export const lanaLog = ({ message, e = '' }) => {
+// Common used LANA logs tags:
+// info - network issues
+// warn - authoring related misconfigs or similar
+// error - actual error ( ex. cannot read Y of undefind )
+// <module> - extra tag to help centralise all logs related to a specific module
+
+const composeLanaTags = (tags = ['default']) => {
+  if (
+    !Array.isArray(tags)
+    || tags.length === 0
+    || tags.some((t) => typeof t !== 'string')
+  ) {
+    return ['default'];
+  }
+  return tags;
+};
+
+export const lanaLog = ({ message, e = '', tags }) => {
```

Comment:
If we remove the `composeLanaTags` method you could just set 
```suggestion
export const lanaLog = ({ message, e = '', tags = ['default'] }) => {
```